packages/ocamlformat/ocamlformat.v0.2/opam

```diff
@@ -0,0 +1,21 @@
+opam-version: "1.2"
+maintainer: "OCamlFormat Team <ocamlformat-team@fb.com>"
+authors: "Josh Berdine <jjb@fb.com>"
+homepage: "https://github.com/ocaml-ppx/ocamlformat"
+bug-reports: "https://github.com/ocaml-ppx/ocamlformat/issues"
+license: "MIT"
+dev-repo: "https://github.com/ocaml-ppx/ocamlformat.git"
+build: [
+  ["jbuilder" "subst"] {pinned}
+  [make]
+]
+depends: [
+  "base"
+  "base-unix"
+  "cmdliner"
+  "jbuilder" {build}
+  "ocaml-migrate-parsetree" {>= "1.0.5"}
+  "ocamlformat_support"
```

Comment:
do you want to constrain this ocamlformat_support to v0.2 as well, to make sure they are in sync?

Reply:
I expect that the support package will need to be updated much less frequently, and was thinking that in the future, ocamlformat could add a version constraint on an updated ocamlformat_support package if necessary. Perhaps I have misunderstood or overlooked something though?

Reply:
Your understanding is spot on. We don't add constraints unless they are needed to make pinning easier.  Thanks for clarifying!